app/src/main/res/layout/activity_main.xml

```diff
@@ -2,6 +2,7 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
```

Comment:
Don't see why we need to restrict this vertical orientation. Any reason why this was added?

Reply:
We don't. This was my mistake. I was playing around activity_main.xml. I can remove this; however, because waveView is occupying full width and height, declaring vertical would not change anything

Reply:
Ah, okay. Do you mind removing it? That would not allow the activity to rotate when the orientation is flipped, right?

Reply:
It is removed now.

FYI,  android:orientation = "vertical" is not associated with activity rotation. It describes how its children views are placed. Take a look at the first picture of this link. https://www.tutorialspoint.com/android/android_linear_layout.htm

Since the WaveView is occupying all the width and height (android:width = "match_parent" and android:height = "match_parent) and is the only child in that Viewgroup (LinearLayout), declaring to vertical or horizontal does not affect  how WaveView is drawn.

Reply:
ah, gotcha! Thanks!